routes/web.php

```diff
@@ -45,24 +47,32 @@
     // Authenticate exclude index page
     Route::group(['middleware'=>'auth'], function () {
 
-        Route::get('/admin/dashboard', function () {
-            return view('admin/dashboard');
-        });
-        Route::get('/admin/dashboard_all_users', function () {
-            return view('/admin/dashboard_all_users');
-        });
-        Route::get('admin/all_restaurants', function () {
-            return view('admin/all_restaurants');
-        });
-        Route::get('admin/all_reviews', function () {
-            return view('admin/all_reviews');
-        });
-        Route::get('/admin/dashboard', function () {
-            return view('/admin/dashboard');
-        });
-        Route::get('admin/dashboard_all_owners', function () {
-            return view('admin/dashboard_all_owners');
-        });
+        Route::get('/admin/dashboard', [App\Http\Controllers\AdminController::class, 'index'])->name('admin.dashboard');
+        // Route::get('/admin/dashboard', function () {
```

Comment:
we can remove the commented codes here now since we do not need them anymore.

Reply:
I deleted them!!